File: src/main/java/io/github/rcarlosdasilva/weixin/model/response/user/tag/bean/UserTag.java
package io.github.rcarlosdasilva.weixin.model.response.user.tag.bean;

public class UserTag {

  private int id;
  private String name;
  private int count;

  /**
   * 标签id.
   * 
   * @return id
   */
  public int getId() {
    return id;
  }

  /**
   * 标签名.
   * 
   * @return name
   */
  public String getName() {
    return name;
  }

  /**
   * 标签下用户数.
   * 
   * @return count
   */
  public int getCount() {
    return count;
  }

}

File: src/main/kotlin/io/github/rcarlosdasilva/weixin/model/response/UserTag.kt
package io.github.rcarlosdasilva.weixin.model.response

import com.google.gson.annotations.SerializedName

class UserTag {
  /**
   * 标签id
   */
  val id: Int = 0
  /**
   * 标签名
   */
  val name: String? = null
  /**
   * 标签下用户数
   */
  val count: Int = 0

}

class UserTagCreateResponse {
  val tag: UserTag? = null
}

class UserTagListResponse {
  val tags: List<UserTag>? = null
}

class UserTagListBasedUserResposne {
  @SerializedName("tagid_list")
  val tags: List<Int>? = null
}